specification/metrics/listunitprice.md

```diff
@@ -1,8 +1,8 @@
 # List Unit Price
 
-The List Unit Price represents the suggested provider published unit price for a single pricing measurement unit, which incorporates volume/tier-based rates while excluding any negotiated or commitment-based discounts. This price is denominated in the [Billing Currency](#billingcurrency). It is used to calculate List Cost and savings based on rate optimization activities.
+The List Unit Price represents a suggested provider-published unit price for a single pricing measurement unit (identified by a [Sku Price Id](#skupriceid)), which incorporates free-tier, volume/tier-based, BYOL-based and dynamically priced SKU rates, while excluding any other discounts or the amortization of upfront charges (one-time or recurring). This price is denominated in the [Billing Currency](#billingcurrency). The List Unit Price can be used for calculating savings based on rate optimization activities. In this context, activities such as increased consumption leading to lower volume/tier-based rates, the use of pre-owned software licenses (BYOL - Bring Your Own License), leveraging interruptible resources and/or services, or optimizing usage to take advantage of dynamic pricing models are not considered rate optimization activities.
 
```

Comment:
We need an explicit statement about the pricing currency issue in all price columns. This is too big of an issue to brush under the rug.
```suggestion

While FOCUS includes all prices in the BillingCurrency, providers may use a separate currency for pricing and invoicing. This means prices in FOCUS may not be the same as what is in the provider's cost and usage data, price list, or invoice. Practitioners need to account for the provider's exchange rate when comparing prices in FOCUS to other sources.

```

Reply:
Will cover this in a separate PR